app/controllers/admin/branches_controller.rb

```diff
@@ -32,12 +37,11 @@ def update
 
   def destroy
     @branch.destroy ? flash[:notice] = t('.success') : flash[:alert] = t('.failure')
-
-    redirect_to branches_url
+    redirect_to branches_path
   end
 
   def meals
-    @branch_meals = @branch.branches_meals.includes(:meal, :branch).order(:id)
+    @branch_meals = @branch.branches_meals.includes(:meal, :branch).order(:placed_on)
```

Comment:
`placed_on ` does not exists on branch meals.

Reply:
By mistake updated this. Fixed now